src/components/Dialog/index.web.tsx

```diff
@@ -217,14 +217,13 @@ export const InnerFlatList = React.forwardRef<
     <Inner
       label={label}
       style={[
+        a.overflow_hidden,
+        a.px_0,
         // @ts-ignore web only -sfn
-        {
-          paddingHorizontal: 0,
-          maxHeight: 'calc(-36px + 100vh)',
-          overflow: 'hidden',
-        },
+        {maxHeight: 'calc(-36px + 100vh)'},
         webInnerStyle,
-      ]}>
+      ]}
+      contentContainerStyle={[a.px_0]}>
```

Comment:
This is the main line, the rest are cleanups

Reply:
Seems good, I think we want to also pass in the `contentContainerStyle` from props here too though.